posit_workbench_login_guidance.md

```diff
@@ -0,0 +1,39 @@
+# Guidance to log in and start a session on Posit Workbench
+
+Original Author: Ciara Gribben
+
+Last Updated: December 2022
+
+## Logging in to Posit Workbench
+1. Ensure you are connected to the VPN.
+
+2. Open a web browser (preferably Microsoft Edge).
+
+3. Navigate to Posit Workbench at: https://pwb.publichealthscotland.org/auth-sign-in?appUri=%2F
```

Comment:
Can the URL be changed to [https://pwb.publichealthscotland.org/](https://pwb.publichealthscotland.org/)

Reply:
Done!